Maquetado/scss/details.scss

```diff
@@ -49,26 +71,54 @@
 }
 
 .book-title {
-  font-size: 32px;
-  font-weight: 700;
-  line-height: 38px;
+  @include text-format(32px, 700, 38px);
   margin-right: 5px;
 }
 
 .book-gender {
-  font-size: 24px;
-  font-weight: 700;
-  line-height: 28px;
+  @include text-format(24px, 700, 28px);
 }
 
 .item-key {
-  font-size: 20px;
-  font-weight: 700;
-  line-height: 24px;
+  @include text-format(20px, 700, 24px);
   margin-right: 5px;
 }
 
 .item-value {
   font-size: 20px;
   line-height: 24px;
 }
+
+.list {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 0 auto;
+  max-width: 1000px;
```

Comment:
Remember assign `width: 100%` for the responsive

Reply:
Please fix branch Conflicts before the push